### ISSUE TITLE
Remove EVA capability from Mercury/Vostok

### ISSUE THREAD

**Issue by rnyoakum** [raceintospace/raceintospace#393]
From @peyre in https://github.com/raceintospace/raceintospace/issues/216#issuecomment-531840374:
> Mercury and Vostok have one significant inaccuracy: you shouldn't be able to do EVA with them. Mercury and Vostok have one significant inaccuracy: you shouldn't be able to do EVA with them. However, removing that capability would make it so there's very little you can do with that capsule; if we removed it I would want to create another mission for it, maybe a Medical Space Tests mission or something like that, to give you a reason to fly more than two or three missions with the capsule.

I'm wondering if this is something that's worth examining, for historical accuracy. My analysis of EVA is that
- EVA is comparatively cheap to research,
- EVA is very safe from R&D alone,
- EVA has no milestone penalty associated with it
so it's a prestige goal that often won't merit its own mission. That is, it's safe enough that every manned mission gets +EVA tacked on until the EVA goal is achieved. For Mercury/Vostok, that means _all_ Earth Orbitals becomes EO+EVA and all Duration Bs becomes Duration B + EVA. Furthermore, the Mercury/Vostok is so dangerous that planning a separate mission for only the Spacewalk prestige goal is daunting (20%+ failure on 4+ Capsule tests = Yikes!). In my experience, the AI packages EVA with the first mission it can.

For those reasons, I feel the EVA doesn't get enough respect. If the AI spacewalks first, you push EVA back to your Lunar capsule. Otherwise, it's a secondary goal that gets tacked on to everything else. Maybe that it's fate, or maybe the extra safety of the multi-person manned programs would make the EVA a worthy goal on it's own. I confess I've spent far more time programming RIS than playing it, so I'd be interested in other people's experiences.

As for the second part of the remarks,
>However, removing that capability would make it so there's very little you can do with that capsule; if we removed it I would want to create another mission for it, maybe a Medical Space Tests mission or something like that, to give you a reason to fly more than two or three missions with the capsule.

I've always found the Mercury/Vostok a statistical death trap, and the quicker done with it the better. Historically, there were six manned Mercury flights, so 2-3 missions seems reasonable considering RIS makes it virtually (literally?) impossible to have a launch schedule as vigorous as NASA.

Thoughts?

**Comment by peyre** [comment]
I agree that EVA feels shortchanged in the game.  I've thought before that a system of EVA points could redress that imbalance.  It has the complication that it would introduce...a complication in the game, and there's no equivalent to the LM building to display the number of EVA points.  I suppose we could show the number in R&D and Purchasing.  Another idea that's come to mind is maybe lowering the Max R&D for EVA suits--maybe even lowering it by something like 10 points and giving +2 for every mission that successfully does an EVA test.  That, obviously, entails a complication of its own.

Removing EVA capability from Mercury/Vostok would make the game more true to history.  As I recall, the astronauts really had to squeeze into the Mercury capsule, so I doubt an EVA would be realistic for it.  But again, there's so little you can do with Mercury/Vostok, I hate to take it away unless we offer something to replace it.  My suggestion in #210 of adding a Medical Space Tests mission (or something like that) has the problem that you can't just add missions to the game, just like that.  Missions are numbered and there aren't any gaps in them (say, if mission 12 were missing we could stick something in its place).  Another option is to have some 1-man mission equivalent to LM Points: you suffer -6 to each capsule stage of a multi-manned mission unless you've flown 3 single-seat missions, etc.  That could be a nightmare to implement like the EVA point system above, and I like the EVA points idea better than the 1-person capsule point idea.

**Comment by peyre** [comment]
Ryan recently discovered two missions that seem to be duplicates, 30 and 35 - and they both seem to be duplicates of mission 23.  One of the former could become a Joint Synchronized Orbit (or something like that), where two spacecraft fly close together in Earth orbit, like Soyuz 3/4 and 5/6.  That would be something that Mercury/Vostok could do in place of EVA.  Its prestige value could be similar to Spacewalk and Manned Docking: 8 for a first, 4 for a second, 1 after, and -3 for failures.

**Comment by peyre** [comment]
I suggested recently (or thought about and forgot to suggest; I can't seem to find it now) that we change how EVA Safety is handled.  As it is, where you could have them at Max R&D and take them on the lunar landing without ever using them first.  That is, EVA missions are effectively optional.  I think we may have discussed working out some kind of EVA Point system similar to LM Points, but seems to me a better model would be to have them work similarly to DMs rather than LMs.

Maybe EVA Suits could be researchable up to a point, say 65%.  Then you would have to test them in space to improve their Safety, gaining 5% on a failure and 10% on a successful spacewalk.  EVA missions would need to be heavily weighted against catastrophic failure, much like Docking tests.  Also, in the event of a cat failure, maybe suits could be researched back to where you left off.
